daemon/import.go

```diff
@@ -23,13 +24,20 @@ import (
 // inConfig (if src is "-"), or from a URI specified in src. Progress output is
 // written to outStream. Repository and tag names can optionally be given in
 // the repo and tag arguments, respectively.
-func (daemon *Daemon) ImportImage(src string, newRef reference.Named, msg string, inConfig io.ReadCloser, outStream io.Writer, config *container.Config) error {
+func (daemon *Daemon) ImportImage(src string, newRef reference.Named, msg string, inConfig io.ReadCloser, outStream io.Writer, changes []string) error {
 	var (
 		sf   = streamformatter.NewJSONStreamFormatter()
 		rc   io.ReadCloser
 		resp *http.Response
 	)
 
+	// 'err' MUST NOT be defined within this block, we need any error
```

Comment:
comment didn't need to move. applicable only to the context of the original block.